corehq/apps/app_manager/models.py

```diff
@@ -3258,7 +3258,7 @@ def create_profile(self, is_odk=False, with_media=False, template='app_manager/p
             'uniqueid': self.copy_of or self.id,
             'name': self.name,
             'descriptor': u"Profile File"
-        }).decode('utf-8')
+        }).encode('utf-8')
```

Comment:
Yeah. I consulted with @millerdev while looking at this , because I knew he would be the expert. Feel free to wait for him to comment here though.